usdt-impl/src/stap3.rs

```diff
@@ -0,0 +1,184 @@
+//! The SystemTap probe version 3 of the USDT crate.
+//!
+//! Used on Linux platforms without DTrace.
+
+// Copyright 2021 Oxide Computer Company
```

Comment:
I think this is probably copy-paste. I'm actually not sure _who_ owns the copyright here (probably you, but IANAL), but the year should be current in any case.

Reply:
A lot of this file is copied over from empty and non-linker, so I kept this as Oxide Computer. The argument handling code I moved to a different file where I put the copyright as mine; I don't really give a hoot about that so if you decide you'd prefer it as Oxide Computer then I'll happily change it.